### DISCUSSION TITLE
custom buttons not showing

### DISCUSSION THREAD

**Discussion by suziuk** [arkypita/LaserGRBL#1945] [General]
Hi I am new to this and if I'm honest, confused!  I followed instructions on downloading custom buttons and it goes through the motions but the buttons still do not become available.  I currently have a laser machine I cant seem to use.  Any help gratefully received thanks.  Be kind!

**Comment by Tribo-2dot0** [comment]
Se a janela estiver muito pequena os botões ficam escondidos, expanda a janela.
